lib/ansible/plugins/action/pause.py

```diff
@@ -102,8 +104,8 @@ def run(self, tmp=None, task_vars=None):
         # Begin the hard work!
 
         start = time.time()
-        result['start'] = str(datetime.datetime.now())
-        result['user_input'] = ''
+        result['start'] = to_text(datetime.datetime.now())
+        result['user_input'] = b''
```

Comment:
ActionModule.run() results only need to be pickle'able? Or do they need to be JSON serializable?  The plain[1] py3 json.dumps() doesn't understand bytestrings, but not sure if that is an issue here.